essence-of-live-coding-vivid/src/LiveCoding/Vivid.hs

```diff
@@ -58,7 +58,6 @@ vividHandleParametrised = ParametrisedHandle { .. }
     -- Synthdef or start/stop state changed, need to release and reinitialise
     changeParametrised old new synth = defaultChange createParametrised destroyParametrised old new synth
 
-deriving instance Eq (SynthDef args)
```

Comment:
Why this change?

Reply:
Nevermind, I didn't read your message completely it seems.

Reply:
vivid was not building anymore with the version of hashable picked by cabal because it needed SynthDef to be an instance of `Eq`. I made a commit to fix that, and sent a pull-request to Tom, which he accepted: https://git.sr.ht/~amindfv/vivid/commit/2b340aa5c57dc5a0d57e55943b38fc92c9c882e3 . He then bumped vivid to 0.5.2.0.

Since the Eq instance is now in Vivid it is not needed here.